corehq/form_processor/models.py

```diff
@@ -258,9 +259,7 @@ class AbstractAttachment(models.Model):
 
     @property
     def filepath(self):
-        if getattr(settings, 'IS_TRAVIS', False):
-            return os.path.join('/home/travis/', str(self.attachment_id))
-        return os.path.join('/tmp/', str(self.attachment_id))
+        return os.path.join(gettempdir(), str(self.attachment_id))
```

Comment:
Here's what I gather from the output:

- `gettempdir()` did not pay attention to the `TEMPDIR=$HOME/tmp` environment variable setting. The reason why is not clear. Maybe environment variables are not set in the `install` section (seems surprising). In hindsight I wish I'd checked the environment in the python script to verify if it was set or not.
- `gettempdir()` returned `/tmp` and the script was able to write to that directory. This seems to contradict what you've been saying about `/tmp` not being writeable. Does this seem surprising to you?
- `/home/travis/tmp` does not exist. Not surprising, we didn't create it. I just wanted to verify that it was not being created somewhere else that I didn't see.

Given these findings I'm inclined to continue to use `gettempdir()` -> `/tmp` in `AbstractAttachment.filepath` rather than trying to do something special when running in a travis environment. Does that seem reasonable to you?